### PR TITLE
Search on layout

### PR DESCRIPTION
**Pull request by gleono**
Moved the search input into the layout.html
Search works by sending the typed information to `api/search` I don't know why it also sends `term` as a parameter but is ignored by our service lol.

I added some tags to the view-single, feel free to remove them or style them better. I was testing searching using tags.

### CODE REVIEW

File: services/search.py [hunk 1]
```diff
@@ -16,18 +16,23 @@ def get(self):
 
         s = urllib.unquote_plus(s)
 
+        query = '"{0}"'.format(s.replace('"', ''))
```
Comment (gleono) [inline services/search.py:19]: I was finding with tags using this query. I had to modify it because it was crashing typing commas.

Comment (rayolanderos) [reply services/search.py:19]: Ok. I have a feeling the tag bug was introduced in this file so I'll debug.

File: services/search.py [hunk 2]
```diff
@@ -16,18 +16,23 @@ def get(self):
...
         stream_list = []
 
         expr_list = [search.SortExpression(
             expression='name', default_value='',
-            direction=search.SortExpression.DESCENDING)]
+            direction=search.SortExpression.DESCENDING), 
+            search.SortExpression(
```
Comment (gleono) [inline services/search.py:25]: I also added this one, I think I was finding tags with this, but not sure anymore. Feel free to remove it.

File: www/view-single.html [hunk 3]
```diff
@@ -12,6 +12,12 @@ <h1 class="text-center text-lg-left">{{ stream.name }} <a id="btn-geo-view" clas
           </div>
           {% endfor %}
         </div>
+        <div class="bootstrap-tagsinput">
+          {% for tag in stream.tags %}
+          <span class="tag label label-info">{{ tag }}</span>
```
Comment (rayolanderos) [inline www/view-single.html:17]: I'll take a look at the styling. Worse case scenario we keep it for debugging and css hide it.

**Review by rayolanderos — APPROVED** [review]
Looks great!

File: www/js/search.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+        select: function (event, ui) {
+            window.location = 'view?id='+ ui.item.id;
+        }
+    });
+
+    autoComplete.data('ui-autocomplete')._renderItem = function (ul, item) {
+        var inner_html = '<div class="list_item_container"><img src="' + item.cover_url + '" />' + item.name + '</div>';
+        return $('<li></li>').attr('data-value', item.id).append(inner_html).appendTo(ul);
+    };
+}());
```
Comment (gwoodard3) [inline www/js/search.js:27]: This was probably where I was messing up my code.  The example on the site has this code in an html file so was trying to put this code in the html file

Comment (rayolanderos) [reply www/js/search.js:27]: Most of the times you'll start with HTML and then dynamically update it through the script. Your example was probably starting off with something built already.

**Pull request merged**